src/Stmt.cc

```diff
@@ -110,6 +111,11 @@ bool Stmt::IsPure() const
 	}
 
 void Stmt::Describe(ODesc* d) const
+	{
+	StmtDescribe(d);
```

Comment:
Why the rename of all the derived methods when the new top-level just does a direct call to the same thing?

Reply:
The statement body used to read:
``` 
        if ( d->DoOrig() )
                Original()->StmtDescribe(d);
        else
                StmtDescribe(d);
```
but @jsiwek was not a fan of `ODesc` knowing about the notion of "original".  I'd like to leave this factoring in place, though, because I'm expecting that something like this (= wrapping `Stmt` descriptions with some refinements) will be needed once we progress to the AST transformations that are in the future.